src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -265,6 +265,10 @@
   develop(bool, OptoCoalesce, true,                                         \
           "Use Conservative Copy Coalescing in the Register Allocator")     \
                                                                             \
+  product(uint, IFGEdgesLimit, 10000000, DIAGNOSTIC,                        \
+          "Maximum allowed edges in interference graphs")                   \
```

Comment:
```suggestion
          "Maximum allowed edges in the interference graphs")               \
```
Nit: usually, the flag descriptions use "the"